src/serac/physics/thermal_conduction.hpp

```diff
@@ -199,6 +199,24 @@ class ThermalConduction : public BasePhysics {
    */
   virtual ~ThermalConduction() = default;
 
+  /**
+   * @brief Get an mfem::Operator that calculates the residual at the current outputState
+   */
+  [[noreturn]] mfem::Vector outputResidual() override
```

Comment:
tagging @white238 and @joshessman-llnl , Any thoughts on attributes and throwing errors?

Reply:
This should be removed with the `detail` suggestion changes.